finetuner-workflow/finetune-workflow.yaml

```diff
@@ -242,6 +254,50 @@ spec:
               values:
               - "{{workflow.parameters.region}}"
 
+  - name: dataset-downloader
+    inputs:
+      parameters:
+        - name: output
+    retryStrategy:
+      limit: 1
+    # The dataset downloader runs as the nonroot user so the dataset folder in the PVC
+    # needs the correct permissions.
+    initContainers:
+      - name: dataset-perms
```

Comment:
fyi, don't name your init container `init` in argo workflows since that overrides the container named `init` that argo adds under the hood to set things up.

Reply:
lol :)